src/server/test/web/readingsCompareMeterFlow.js

```diff
@@ -11,7 +11,8 @@ const Unit = require('../../models/Unit');
 const { prepareTest,
 	expectCompareToEqualExpected,
 	getUnitId,
-	METER_ID } = require('../../util/readingsUtils');
+	METER_ID, 
+	unitDatakWh} = require('../../util/readingsUtils');
```

Comment:
This file is for flow data so this should not be needed. A search found it is not used anywhere so it should be removed.